Common/include/option_structure.hpp

```diff
@@ -217,14 +217,25 @@ enum ENUM_SOLVER {
   DISC_ADJ_EULER = 35,
   DISC_ADJ_RANS = 36,
   DISC_ADJ_NAVIER_STOKES = 37,
-  DISC_ADJ_FEM = 40
+  DISC_ADJ_DG_EULER = 65,
```

Comment:
For consistency, DISC_ADJ_DG_EULER -> DISC_ADJ_FEM_EULER, since FEM_EULER is the primal